numpy/doc/glossary.py

```diff
@@ -416,6 +428,12 @@
        Universal function.  A fast element-wise array operation.  Examples include
        ``add``, ``sin`` and ``logical_or``.
 
+   vectorized
+       A loop-based function that operates on data with fixed strides.
```

Comment:
A little weird to have an adjective as a heading with the description of a noun below.

Reply:
improved, maybe